core/src/main/scala/org/apache/spark/scheduler/CoarseCookSchedulerBackend.scala

```diff
@@ -183,6 +195,9 @@ class CoarseCookSchedulerBackend(
     import CoarseCookSchedulerBackend.fetchURI
 
     val jobId = UUID.randomUUID()
+    val taskId = sparkMesosScheduler.newMesosTaskId()
+    taskOrExecutorIdToJobId += taskId -> jobId
```

Comment:
Can we log this mapping. It's super useful to have the actual mesos task ID and the corresponding executor ID when trying to match it up with cook logs.